compilers/concrete-optimizer/concrete-optimizer/src/dag/unparametrized.rs

```diff
@@ -392,6 +423,43 @@ impl<'dag> DagBuilder<'dag> {
     }
 }
 
+#[derive(Clone, PartialEq, Debug, Default)]
+pub(crate) struct CompositionRules(HashMap<OperatorIndex, Vec<OperatorIndex>>);
```

Comment:
I guess this is a matter of taste. I don't mind the `.0` since to me it very clearly signals that this is a newtype, and the value really concerns only one thing, which is accessible by the field `0`. At the contrary, I find that giving a name to a one field struct unnecessarily verbose. And if we want to add something later on, we can change the newtype to a struct in one action with `rust-analyzer`.